priv/.credo.exs

```diff
@@ -3,7 +3,7 @@
     %{
       name: "default",
       checks: [
-        # TODO: Add custom checks module
+        {CompassCredoPlugin.Checks.DefdelegateOrder, []}
```

Comment:
```suggestion
        {CompassCredoPlugin.Check.DefdelegateOrder, []}
```
Don't forget in this place too 😛

Reply:
Updated in [9853a6e](https://github.com/nimblehq/compass-credo-plugin/pull/10/commits/9853a6ecac8307917297512ca42d711960509399)